### DISCUSSION TITLE
Large paths on Windows crashed the edition

### DISCUSSION THREAD

**Discussion by RafaelGB** [RafaelGB/obsidian-db-folder#104] [Q&A] [answered]
To address the problem on windows with large paths and close evidence. I will try to give you support with it when I can reproduce it on a windows

**Comment by ObsidianTTRPGProject** [comment]
If it helps I have my vault on GitHub should you need something to test on. 
I've reworked my whole vault now with a max file length of 183 chars and it's still having the same issue. Windows can handle up to 260. 
There certainly looks to be some confusion in the file paths. The Monk folder does not exist in my vault. It was set as an option in a choice field so maybe pulling it from there somehow. 

![image](https://user-images.githubusercontent.com/96321415/172033469-ef58ab1c-f364-4579-9118-371229f8276b.png)

**Reply by RafaelGB** [reply]
could you post your github url? this week I will try to test it with another laptop woth windows

**Reply by ObsidianTTRPGProject** [reply]
I've sent you an invite. 
The main branch has the version with the content that has been modified to reduce the file length. 
There's a backup branch [Backup05062022] that contains the version with really long file paths. 
It's not a small vault, 1.89GB. It doesn't like to be opened in WinZip but 7Zip works fine. 

Thanks mate, appreciate any help you can offer.

**Comment by RafaelGB** [comment]
I accepted it. Lets see if we can catch the problem 😄

Un saludo
________________________________

**Comment by RafaelGB** [comment]
I could reproduce it with your Vault. Looks like a problem with the query associated to the name of the folder. WIth another sources as tags can load 3000+ notes (huge Vault BTW, could I use it as performance testing?)

I will try to reproduce this week at home and debug whats the problem with the name

**Reply by ObsidianTTRPGProject** [reply]
By all means. It will continue to grow so test away :)

**Comment by RafaelGB** [comment]
<img width="1053" alt="Screenshot 2022-06-07 at 13 56 31" src="https://user-images.githubusercontent.com/11924043/172373338-5dff59f0-9796-4b17-aaca-cbcc6c70f495.png">

The query used by the plugin seems not to work with your vault, Looks more like data view bug searching by folder. Inside "The Party" folder even renaming it does not work

**Comment by RafaelGB** [comment, marked as answer]
```dataview
TABLE file
FROM "0. The Party"
```
```dataview
TABLE file
FROM "z_Templates"
```

The first one obtain 0 hits, the second one 130

something inside first breaks data view

**Reply by ObsidianTTRPGProject** [reply]
Thank you. I came to a similar conclusion last night  trying to get the heat map calendar plugin to work I noticed that the code works perfectly when using it in a different vault but mine just throws non stop issues. Given both this plugin and that one rely on dataview I was leaning towards there been something there that is blocking it.

Now to commence the hunt and try figure out what is. 

Thanks for your help with this. I'll come running back to database folder as soon as I figure it out.

**Reply by ObsidianTTRPGProject** [reply]
And tonight things are just working like they should... #pullingmyhairout.

**Reply by RafaelGB** [reply]
there were like 5,6 minor versions of data view, maybe that fix the problem

**Reply by ObsidianTTRPGProject** [reply]
Yeah I suspect as much. It also seems to have solved the issues with 'Heatmap Calendar'. Whatever it was it worked so I'll take it. Thanks for your help mate.

**Reply by RafaelGB** [reply]
anytime!

**Reply by ObsidianTTRPGProject** [reply]
And in case it's of interest I've made a video about your plugin to show it off to the TTRPG crowd. 
https://youtu.be/2v7LO64-C08

**Reply by RafaelGB** [reply]
🫶 thats awesome!
we will try to bring the 1.7.0 this week

https://github.com/RafaelGB/obsidian-db-folder/blob/master/docs/docs/changelog.md

**Comment by RafaelGB** [comment]
Tested on MacOS & Windows OS with same result
